modules/ROOT/pages/managing-business-groups.adoc

```diff
@@ -49,8 +185,8 @@ include::partial$include-nav-organizations.adoc[]
 
 [NOTE]
 --
-* For newer Anypoint Platform accounts, the client ID and secret apply to environments rather than organizations. See xref:environments.adoc[Environments].
-* To change the client secret for an environment or organization, see https://help.mulesoft.com/s/article/How-to-set-or-reset-environment-client-secret-in-Anypoint-Platform[this help article], or contact your customer support representative.
+* For newer Anypoint Platform accounts, the client ID and secret apply to environments rather than business groups. See xref:environments.adoc[Environments].
```

Comment:
```suggestion
* For newer Anypoint Platform accounts, the client ID and secret apply to environments rather than business groups. See xref:environments.adoc[].
```